src/lang/english.txt

```diff
@@ -1440,6 +1440,8 @@ STR_CONFIG_SETTING_PERSISTENT_BUILDINGTOOLS                     :Keep building t
 STR_CONFIG_SETTING_PERSISTENT_BUILDINGTOOLS_HELPTEXT            :Keep the building tools for bridges, tunnels, etc. open after use
 STR_CONFIG_SETTING_EXPENSES_LAYOUT                              :Group expenses in company finance window: {STRING2}
 STR_CONFIG_SETTING_EXPENSES_LAYOUT_HELPTEXT                     :Define the layout for the company expenses window
+STR_CONFIG_SETTING_AUTO_REMOVE_SIGNALS                          :Automatically remove signals during rail construction: {STRING2}
+STR_CONFIG_SETTING_AUTO_REMOVE_SIGNALS_HELPTEXT                 :Automatically remove signals during rail construction if the signals would normally prevent the rail from being constructed
```

Comment:
There should probably be a warning in here thath removing signals can cause train crashes.

Reply:
Good point, added it. I also created a translation for my native language, Dutch.

Reply:
> I also created a translation for my native language, Dutch.

Translations are handled via eints (translator service), not pull requests.

Reply:
Ok, I didn't know that, thanks. Should I strip it from the commit?

Reply:
If you have an account at Eints (translator service), you may be able to make the change there. Else, try contacting the maintainers of OpenTTD's Dutch translation to tell them about that change. Regardless, you should remove the foreign translation changes from this PR.

Reply:
Removed it. I don't have an EInts account, but I'll try to contact them on the forums.